app/javascript/crayons/Modal/__stories__/Modal.stories.jsx

```diff
@@ -1,38 +1,53 @@
 import { h } from 'preact';
 import { useState } from 'preact/hooks';
-import { withKnobs, text, boolean, select } from '@storybook/addon-knobs';
 import notes from './modals.md';
-import { Modal, Button } from '@crayons';
-import '../../storybook-utilities/designSystem.scss';
+import { Modal, ButtonNew as Button } from '@crayons';
 
 export default {
   title: 'Components/Modals',
-  decorator: [withKnobs],
-  parameters: { notes },
+  parameters: {
+    docs: {
+      page: notes,
+    },
+  },
+  argTypes: {
+    size: {
+      control: {
+        type: 'select',
+        options: {
+          default: 'default',
+          small: 's',
+          medium: 'm',
+        },
```

Comment:
Doesn't have to be for now, but I noticed a warning in the console:

```
'control.options' is deprecated and will be removed in Storybook 7.0. Define 'options' directly on the argType instead, and use 'control.labels' for custom labels.
```

Reply:
interesting... yea, i'll definitely take a look at it in some future PR if that's ok, since we may need to update it in few more places afair